packages/editor/src/components/widgets/table/table/Table.css

```diff
@@ -3,6 +3,11 @@
 }
 .table-root .icon-input {
   margin-bottom: var(--size-1);
+  position: sticky;
```

Comment:
Are you sure this behaves always correctly?
Did you test also mapping table search inputs?